rl_algorithms/common/env/utils.py

```diff
@@ -31,18 +30,18 @@ def set_env(
         for env_wrapper in env_wrappers:
             env = env_wrapper(env)
 
-    return env
+    return env, max_episode_steps
 
 
 def env_generator(
-    env_name: str, args: argparse.Namespace, env_wrappers: List[gym.Wrapper] = None
+    env_name: str, max_episode_steps: int, env_wrappers: List[gym.Wrapper] = None
 ) -> Callable:
     """Return env creating function (with normalizers)."""
 
     def _thunk(rank: int):
         env = gym.make(env_name)
-        env.seed(args.seed + rank + 1)
-        env = set_env(env, args, env_wrappers)
+        env.seed(777 + rank + 1)
```

Comment:
seed is fixed.

Reply:
Since it's hard to parse environment seed in ppo setting We decided to leave it for the moment.